src/OrleansCounterControl/Microsoft.Orleans.CounterControl.targets

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" DefaultTargets="Build">
```

Comment:
Can't tell, how it will behave, but we decided to not to support VS2015, hance the change there too.